recipes/pseudonetcdf/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pseudonetcdf" %}
+{% set version = "3.0.0" %}
+{% set sha256 = "eb1ee1fa8f16f4a976d0851084b1a353a47f8ece41ab0519c95358dd53d27942" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/barronh/pseudonetcdf/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+  run:
+    - python
+    - numpy
+    - pandas
+    - netcdf4
+    - scipy
+    - pyyaml
+    - matplotlib
+
+test:
```

Comment:
This recipe looks good to me except for one outstanding question. I see that there is a command line interface to this library with a bunch of tools. Should those be tested?

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

I feel like stuff in `scripts` have perhaps not played well with `noarch: python` due to mangling of `#!/usr/bin/env bash`, but that may be an upstream conda issue. I'll leave it up to you if you want to try to include those tests or if you're fine with just testing the library functionality. After that I'm happy to merge.

Reply:
@synapticarbors - Thanks! In many ways, the scripts are legacy convenience routines. In the next version, many will be removed from the package distribution and added to a separate repo for users. I'd rather not hold up the recipe to test them.

For what its worth, I've had good experience with the scripts on linux and mac. On windows, the performance of the scripts has been more mixed and version dependent. It is really this cross-platform complication that has led me to redesign -- making the scripts less of a central usecase.

Reply:
Sounds reasonable. I'm going to go ahead and merge as is.